.github/workflows/e2e-init.yaml

```diff
@@ -56,15 +56,22 @@ jobs:
           echo "=========Current system kernel================="
           uname -r
 
-      - name: Free disk space
+      - name: Free Disk Space (Ubuntu)
+        # https://github.com/spidernet-io/spiderpool/issues/3277
         # https://github.com/actions/virtual-environments/issues/709
-        run: |
-          echo "=========original CI disk space"
-          df -h
-          sudo rm -rf "/usr/local/share/boost"
-          sudo rm -rf "$AGENT_TOOLSDIRECTORY"
-          echo "=========after clean up, the left CI disk space"
-          df -h
+        uses: jlumbroso/free-disk-space@main
```

Comment:
is this safe for other parallel job ?

Reply:
All normal after testing
https://github.com/ty-dc/spiderpool/actions/runs/8134885063